package.json

```diff
@@ -43,11 +43,9 @@
         "chance": "1.1.7",
         "eslint-config-fluid": "2.0.0",
         "eslint-plugin-jsdoc": "30.7.8",
```

Comment:
Since `eslint-config-fluid` and `eslint-plugin-jsdoc` have already been dependencies of `fluid-lint-all`, is it necessary to include them again as dependencies of this repo?

Reply:
We recommend for "eslint-config-fluid" that you explicitly also depend on "eslint-plugin-jsdoc" if you use it. The issue is that there are problems with packaging an eslint plugin with the shareable config. See https://github.com/eslint/eslint/issues/3458 They are working on a new config format to address this. See: https://github.com/eslint/eslint/issues/13481

I'm not sure if the fluid-lint-all suffers from similar issues related to "eslint-config-fluid" there is a reference to install it in the [fluid-lint-all README](https://github.com/fluid-project/fluid-lint-all#running-the-checks-using-npx) which I took as meaning that you should separately install it.

Reply:
I see. Thanks for the explanation, @jobara.